nixtlats/data/datasets/m5.py

```diff
@@ -0,0 +1,137 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/data_datasets__m5.ipynb (unless otherwise specified).
+
+__all__ = ['M5']
+
+# Cell
+import os
+from dataclasses import dataclass
+from typing import Dict, List, Optional, Tuple, Union
+
+from numba import njit
```

Comment:
```suggestion
```

Reply:
Solved in https://github.com/Nixtla/nixtlats/pull/63/commits/93a8f5490f54e7387d4964b997d92b5763ab545d